yetanotherco/aligned_layer
# perf (aggregator): dont wait for new block if block has already passed

Depends on https://github.com/yetanotherco/aligned_layer/pull/462

Aggregator if a block has already passed. If it has, it does not wait for the next block before responding

# To test

It's running on stage, locally just check everything works. You can run anvil with a higher block time or with manual blocks to check how aggregator behaves.
